modules/end-user-guide/pages/managing-workspaces-with-apis.adoc

```diff
@@ -1,22 +1,27 @@
 :_content-type: ASSEMBLY
-:navtitle: Managing workspaces with OpenShift APIs
-:description: Managing workspaces with OpenShift APIs
+:navtitle: Managing workspaces with {orch-name} APIs
+:description: Managing workspaces with {orch-name} APIs
 :keywords: api, list workspaces, create workspace, restart workspace, stop workspace, start workspace, remove workspace
 // :page-aliases:
 
 [id="managing-workspaces-with-apis"]
-= Managing workspaces with OpenShift APIs
+= Managing workspaces with {orch-name} APIs
 
-...
+On a {platforms-name} cluster, {prod-short} workspaces are represented as `DevWorkspace` custom resources of the same name.
+As a result, if there is a workspace named `my-workspace` in the {prod-short} dashboard, there is a `DevWorkspace` named `my-workspace` in the user's {orch-namespace} on the {orch-name} cluster.
 
-Consider writing this entire page as a reference. (If so, then rename all the files in `/partials/` and in the `include::` statements below from `proc_` to `ref_`.)
```

Comment:
Would this be recommended in this scenario? If yes, I can edit this PR to do so

Reply:
The procedures look great. Thank you, David!